### PR TITLE
Us5003 add students submit request

### PR DESCRIPTION
**Pull request by Stefce02**

### CODE REVIEW

**Review by yujlin2 — APPROVED** [review]
Nice

**Review by Dragan14 — CHANGES_REQUESTED** [review]

File: src/hooks/useGroupsInCourse.tsx
[the remark this answers is not in the document]
Comment (Stefce02) [reply src/hooks/useGroupsInCourse.tsx:1]: Fixed in bb2b06472d04ea9121bee3e4963a9e65b2ea69cf

File: src/hooks/useSelfGroups.tsx
[the remark this answers is not in the document]
Comment (Stefce02) [reply src/hooks/useSelfGroups.tsx:1]: Fixed in bb2b06472d04ea9121bee3e4963a9e65b2ea69cf

File: src/hooks/useUsersInGroup.tsx
[the remark this answers is not in the document]
Comment (Stefce02) [reply src/hooks/useUsersInGroup.tsx:1]: Fixed in bb2b06472d04ea9121bee3e4963a9e65b2ea69cf

**Review by Dragan14 — APPROVED** [review]
lgtm

**Pull request merged**